### DISCUSSION TITLE
Add sort system field to an existing collection

### DISCUSSION THREAD

**Discussion by sMesle** [directus/directus#5746] [Archive]
Hello,
I have created a collection with data inside. And now I want to add a sort field to it.

I have tried to replicate this field manually on my existing collection, but on the collection detail I can't use the drag and drop functionnality.

Is there a way to add a sort field to an existing collection ?

Thanks :)

**Comment by sMesle** [comment]
Ok I found how to do it,

You can replicate the sort field in your existing collection
Then on the sort category of your collection, simply set the sort field to use :)

**Reply by benhaynes** [reply]
Just added this to help clarify: https://github.com/directus/directus/commit/9687970a32af252ee723a6a1f398a3dd8bef50b9
